site/_posts/2017-03-12-marching-ahead-embedded-ansible.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Last Week in ManageIQ: Marching ahead with the embedded Ansible integration
+author: lgalis
+date: 2017-03-12 21:33:30 -04:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Laura](https://github.com/lgalis) here bringing you this week's edition of Last Week in ManageIQ!
+
+## Featured
+We continue to make great progress as we move closer to the development complete milestone for our Fine release. This week, 
+the team had 293 commits in 184 PRs across our combined repositories: 73 PRs in the [manageiq](manageiq PRs merged) repo,
+84 in the [manageiq-ui-classic](manageiq-ui-classic PRs merged) repo, 17 in the [manageiq-ui-service](manageiq-ui-service PRs merged), 
+with the rest in the various providers' repos.
+ 
+ 
+## Improved
+
+In this [PR](https://github.com/ManageIQ/manageiq/pull/14239), [Jason](https://github.com/Fryguy) moved dynamic method building into DrbRemoteInvoker,
+allowing it to run in the launched automate ruby process directly and not over DRb, preventing random DRb thread issues.
+
+The event collection for the datawarehouse provider now hasa a new field to event streams to keep track of the ext_management_system that carried the event in,  
+as added by [Mooli](https://github.com/moolitayer) [here](https://github.com/ManageIQ/manageiq/pull/12773).
+
+## Fixed
+
+[JoeR](https://github.com/jrafanie) fixed a failure that would occur after starting new puma workers and try to gracefully restart apache.
+In his [PR] (https://github.com/ManageIQ/manageiq/pull/14007), puma workers bound to a port from STARTING_PORT to the maximum worker count 
+port (3000 to 3009 if max  worker count is 10). Configure apache at boot with these ports as balancer members.
+
+[Libor]](https://github.com/lpichler) created a fix in this [PR](https://github.com/ManageIQ/manageiq/pull/14229) to allow regex for a
+MiqExpression::Field containing numbers in associations.
+
+## New
+
+We now support Rails 5.0.2.] (Support Rails 5.0.2).
+
+Check out [Ladas]() Target and Target Collections abstractions [here](#https://github.com/ManageIQ/manageiq/pull/14249).
+
+With this new [PR](https://github.com/ManageIQ/manageiq/pull/14225)  - the Ansible Play Data is collected during a Job run.
+
+The UI for the ems_physical_pages is now available, added by [Julian](https://github.com/juliancheal) in this [PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/196) 
+
+
+## Deleted
+
+[Martin](https://github.com/martinpovolny) uses GenericShowMixin to unify and remove #show functionality from middeware controllers -[PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/590)
+He also started on the work of removing the case logic for setting page title and moving the decision back to the controllers in this first Title Refactoring [PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/520)
```

Comment:
Not sure if the extra `-` was added on purpose. If it was please ignore.

May need a `.` at the end of the sentence.